Documentation/commands.md

```diff
@@ -129,6 +129,27 @@ $ journalctl -M rkt-bc3c1451-2e81-45c6-aeb0-807db44e31b4 -t redis
 
 Additionaly, logs can be programmatically accessed via the [sd-journal API](https://www.freedesktop.org/software/systemd/man/sd-journal.html).
 
+Currently there are known 2 main issues with loggin in rkt:
+* In some rare situations when an application inside the pod is writing to stdout/stderr instead of journal (i.e. nginx) there is no way to obtain logs.
```

Comment:
Those should really be `/dev/stdout` and `/dev/stderr`, without any mention of the journal.